Drift/Private/DriftBase.cpp

```diff
@@ -3359,6 +3359,52 @@ void FDriftBase::RemovePlayerFromMatch(int32 playerID, const FDriftPlayerRemoved
     request->Dispatch();
 }
 
+void FDriftBase::UpdatePlayerInMatch(int32 playerID, const FDriftUpdateMatchPlayerProperties& properties, const FDriftPlayerUpdatedDelegate& delegate)
+{
+    if (state_ != DriftSessionState::Connected)
+    {
+        DRIFT_LOG(Base, Warning, TEXT("Attempting to update player in match without being connected"));
+        delegate.ExecuteIfBound(false);
+        return;
+    }
+
+    DRIFT_LOG(Base, Verbose, TEXT("Updating player '%d' in match '%d'"), playerID, match_info.match_id);
+
+    JsonValue Payload{ rapidjson::kObjectType };
+    if (properties.status.IsSet())
+    {
+        JsonArchive::AddMember(Payload, TEXT("status"), properties.status.GetValue());
+    }
+    if (properties.team_id.IsSet())
+    {
+        JsonArchive::AddMember(Payload, TEXT("team_id"), properties.team_id.GetValue());
+    }
+    if (properties.statistics.IsSet())
+    {
+        JsonArchive::AddMember(Payload, TEXT("statistics"), properties.statistics.GetValue());
+    }
+    if (properties.details.IsSet())
+    {
+        JsonArchive::AddMember(Payload, TEXT("details"), properties.details.GetValue());
+    }
+
+    const FString Url = FString::Printf(TEXT("%s/%i"), *match_info.matchplayers_url, playerID);
```

Comment:
Is the player url not returned in the match data? There's a players API which does return the URLs. We should not generate URLs if it's possible to avoid it.

Reply:
This is copy pasta from removing a player from a match.
When adding a player to a match, a url is indeed returned.

I'll add some internal caching for match players and use that url instead in both Remove and Update.